pkg/token/token.go

```diff
@@ -4,16 +4,11 @@ package token
 
 import (
 	"context"
-	"crypto/sha256"
-	"encoding/base64"
 	"encoding/json"
 	"fmt"
-	"strings"
 	"time"
 
-	"github.com/convox/hid"
-	"github.com/convox/u2f/u2fhid"
-	"github.com/convox/u2f/u2ftoken"
+	"github.com/ddollar/go-u2fhost"
```

Comment:
Needed to change to `ddollar/go-u2fhost`, the same implementation we use on convox/convox (v3 racks), because the convox/hid was incompatible with arm64 arch.